tool/tsh/tsh.go

```diff
@@ -1553,7 +1636,33 @@ func showApps(apps []types.Application, active []tlsca.RouteToApp, verbose bool)
 	}
 }
 
-func showDatabases(clusterFlag string, databases []types.Database, active []tlsca.RouteToDatabase, verbose bool) {
+func showDatabases(clusterFlag string, databases []types.Database, active []tlsca.RouteToDatabase, format string, verbose bool) error {
+	format = strings.ToLower(format)
+	switch format {
+	case teleport.Text:
+		showDatabasesAsText(clusterFlag, databases, active, verbose)
+	case teleport.JSON, teleport.YAML:
+		if databases == nil {
+			databases = []types.Database{}
+		}
```

Comment:
Will Marshal panic is databases are nil?

Reply:
Same as above.